### PR TITLE
Add br.com.eduk.school

### PR DESCRIPTION
**Pull request by greghorymetzker**
### Please confirm your submission meets all the criteria

Please describe your application briefly below.

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by hfiguiere** [comment]
This is missing:

> Please describe your application briefly below.

**Comment by bbhtt** [comment]
Same as https://github.com/flathub/flathub/pull/4954 and you haven't answered any questions there.

File: br.com.eduk.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+app-id: br.com.eduk
```
[the remark this answers is not in the document]
Comment (greghorymetzker) [reply br.com.eduk.yml:1]: I didn't understand how to solve this problem. Could you provide an example to make it easier for me to understand?

**Comment by greghorymetzker** [comment]
> O mesmo que [# 4954](https://github.com/flathub/flathub/pull/4954) e você não respondeu nenhuma pergunta.

I answered all the questions and, in this new version, corrected all the issues raised in #4954

**Comment by greghorymetzker** [comment]
> Está faltando isso:
> 
> > Descreva brevemente sua aplicação abaixo.

eduK is a freemium SaaS educational platform, built with Electron, focused on online vocational courses. It offers over 3,200 courses, adopting a model that combines free access to selected content with subscription options for advanced features. Aimed at professional training, this is evidenced by the issuance of more than 100,000 certificates, and it promotes the professional success of its users by facilitating employment opportunities and business partnerships.

File: br.com.eduk.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (bbhtt) [reply br.com.eduk.yml:1]: Also is there a way to verify, you are associated with https://app.eduk.com.br/ ? Can you please add that website to your github repo or something else?

File: br.com.eduk.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        . ../flatpak-node/electron-builder-arch-args.sh
+        npm run dist -- $ELECTRON_BUILDER_ARCH_ARGS  --linux --dir
+      # Bundle app and dependencies
+      - cp -a dist/linux*unpacked /app/main
+      # Install app wrapper
+      - install -Dm755 -t /app/bin/ ../run.sh
+      - install -Dm444 logo.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm444 flatpak/${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm444 flatpak/${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+    subdir: main
```
Comment (bbhtt) [inline br.com.eduk.yml:45]:
```suggestion
```

Doesn't exist in source repo

**Comment by greghorymetzker** [comment]
> Dito isto, acho que a utilidade deste aplicativo é limitada, pois é essencialmente uma visualização da página da web. Não é um bloqueador, mas provavelmente deveríamos perguntar o que aconteceu com o suporte PWA no software GNOME.

The goal is to enhance the application for offline videos through Electron. Currently, schools in various countries use exclusively Flathub for software management.

**Comment by bbhtt** [comment]
bot, build br.com.eduk

**Comment by cassidyjames** [comment]
Hey admins, for extra context, we're working with eduK at Endless and are planning to ship eduK as an app for some deployments. We considered GNOME Software web app integration (which we have enabled in Endless OS, at least), but as @greghorymetzker mentioned, the goal is to adapt the Electron all into an offline-capable app with offline video playback. Thay’s something GNOME Web currently doesn’t support at all, as the current “web app” support does not support PWAs.

File: br.com.eduk.yml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      #  path: ..
+      #  dest: main
+      - type: git
+        url: https://github.com/edukorg/eduk-electron.git
+        commit: e0e0950184ab5c065ebc17c4b9d37635e0a11425
+      - type: file
+        path: br.com.eduk.desktop
+      - type: file
+        path: br.com.eduk.metainfo.xml
+      - generated-sources.json
```
[the remark this answers is not in the document]
Comment (hfiguiere) [reply br.com.eduk.yml:57]: From the build log

image

[the remark this answers is not in the document]
Comment (greghorymetzker) [reply br.com.eduk.yml:55]: I added the missing files, tested the build locally, and updated the PR. Please run the bot again. Thank you!

**Comment by bbhtt** [comment]
bot, build

File: br.com.eduk.yml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+      - |
+        . flatpak-node/electron-builder-arch-args.sh
+        npm run dist -- $ELECTRON_BUILDER_ARCH_ARGS  --linux --dir
+      # Bundle app and dependencies
+      - cp -a dist/linux*unpacked /app/main
+      # Install app wrapper
+      - install -Dm755 -t /app/bin/ ./run.sh
+      - install -Dm444 logo.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm444 flatpak/${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm444 flatpak/${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
```
Comment (bbhtt) [inline br.com.eduk.yml:44]: https://github.com/edukorg/eduk-electron/blob/main/flatpak/br.com.eduk.metainfo.xml has no screenshots 


See https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#screenshots

Comment (greghorymetzker) [reply br.com.eduk.yml:44]: Added image

Comment (greghorymetzker) [reply br.com.eduk.yml:44]: @bbhtt Could you please run the bot?

File: br.com.eduk.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (hfiguiere) [reply br.com.eduk.yml:1]: this is not a suggestion. it needs to be addressed. The current app-id does NOT meet the requirements

**Comment by greghorymetzker** [comment]
@bbhtt  Could you please run the bot?

**Comment by bbhtt** [comment]
bot, build br.com.eduk

File: br.com.eduk.metainfo.xml
Comment (bbhtt) [inline br.com.eduk.metainfo.xml:1]: Why did you put the file here? Add them to this one and update the commit of eduK.

Comment (greghorymetzker) [reply br.com.eduk.metainfo.xml:1]: Sorry for the lack of attention, I ended up sending it to the wrong place, but I've already solved it and tested it locally. Please, run the bot.

**Comment by bbhtt** [comment]
bot, build br.com.eduk

**Comment by bbhtt** [comment]
App seems fine, the point about ID and ownership still stands https://github.com/flathub/flathub/pull/5000#discussion_r1501978194 and blocks the submission.

Please address those two, when you have the time.

File: br.com.eduk.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (greghorymetzker) [reply br.com.eduk.yml]: @hfiguiere I adjusted the app-id to br.com.eduk.school.

**Comment by greghorymetzker** [comment]
> O aplicativo parece bom, a questão sobre ID e propriedade ainda permanece [# 5000 (comentário)](https://github.com/flathub/flathub/pull/5000#discussion_r1501978194) e bloqueia o envio.
> 
> Por favor, fale com esses dois, quando tiver tempo.

app-id changed and comment answered.

**Comment by bbhtt** [comment]
bot, build br.com.eduk

File: br.com.eduk.yml
Comment (bbhtt) [inline br.com.eduk.yml:1]: The filename here needs to be updated to `br.com.eduk.school.yml`.

**Comment by bbhtt** [comment]
bot, build br.com.eduk.school

**Comment by bbhtt** [comment]
bot, build br.com.eduk.school

**Comment by greghorymetzker** [comment]
@bbhtt Suggestion approved, just start the bot.

**Comment by bbhtt** [comment]
bot, build br.com.eduk.school

**Comment by greghorymetzker** [comment]
@bbhtt I installed it in my local environment, and it worked, we can proceed. Thank you!

screen_success

**Comment by greghorymetzker** [comment]
@barthalion Could you do the merge, please?

**Pull request closed without merging**